src/utils/openmc/statepoint.py

```diff
@@ -7,6 +8,9 @@
 import openmc
 from openmc.constants import *
 
+if sys.version > '3':
```

Comment:
There are always the cool kids and a few that are slower than the rest of the pack in every class.